df.ui.xml

```diff
@@ -354,6 +354,11 @@
             <flag-bit name='first_year'/>
```

Comment:
in order, these are: `FIRSTYEAR`, `EVAL_REQUESTERCANCHECK`, `RUN_SPECIAL_ELECTIONS`, `NEED_TO_DO_TUTORIAL`, `MINOR_VICTORY`, `MAJOR_VICTORY`, `DID_FIRST_CARAVAN_ANNOUNCEMENT`, and `DID_FIRST_CAVERN_ANNOUNCEMENT`

all are prefixed with `PLOTINFOFLAG_` so the type should probably be called` plotinfoflag`, the field is just called `flag`

Reply:
I based the name off of `[CONTEXT:CAVERNS_OPENED]` in `data/vanilla/vanilla_music/objects/music_standard.txt`. Do we want to use the official names for the ones we've already defined, or is that too much hassle?

Reply:
my preference in this regard is to replace our name with the official name if the official name is clearly more "meaningful" than ours (basically, if our name is absent or obviously wrong), and to otherwise keep our name with the official name as a comment otherwise

also if a class structure is not named with the official name the official name should be annotated with an `original-name` attribute.